test/logging.jl

```diff
@@ -249,20 +249,20 @@ end
     # Simple
     @test genmsg(Info, "msg", Main, "some/path.jl", 101) ==
     """
-    I- msg -Info:Main:path.jl:101
+    Info: msg - at Main:path.jl:101
     """
 
     # Multiline message
     @test genmsg(Warn, "line1\nline2", Main, "some/path.jl", 101) ==
     """
-    W- line1
-    |  line2 -Warn:Main:path.jl:101
+    Warn: line1
```

Comment:
I think I'd prefer the original, pre-logging formatting, where info, warning, and error are completely spelled out and in all caps.

Reply:
I would too. This was a quicker change. 

Do you think just the common log levels should be printed uppercase, hardcoded in the `show(::IO, ::LogLevel)` function? Calling `uppercase` when logging would produce `LOGLEVEL(8)` which, unlike the current version, is not the syntax for constructing a LogLevel (plus `uppercase` is now in stdlib so we'd have to add an ASCII uppercase or something).